### PR TITLE
chore(Redesign Servizi): [IARS-29] Add mixpanel track for newly integrated service preference API

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR introduces the tracking ability for the newly implemented APIs for setting single service contact preferences

### CODE REVIEW

File: ts/screens/onboarding/OnboardingServicesPreferenceScreen.tsx
Comment (Undermaken) [inline ts/screens/onboarding/OnboardingServicesPreferenceScreen.tsx:57]:
```suggestion
    // when the user made a choice (the profile is right updated), continue to the next step
```
This comment is not longer valid

File: ts/screens/profile/ServicesPreferenceScreen.tsx
Comment (Undermaken) [inline ts/screens/profile/ServicesPreferenceScreen.tsx:45]:
```suggestion
    // when the user changes his/her preference, track the new preference
    if (
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

**Review by Undermaken — APPROVED** [review]
@fabriziofff waiting for your approval

**Review by Undermaken — APPROVED** [review]

**Review by fabriziofff — APPROVED** [review]

**Pull request merged**
